keyboards/knops/mini/keymaps/mverteuil/keymap.c

```diff
@@ -0,0 +1,349 @@
+};
+
+void led_set_layer(int layer);
+
+/*
+ *   Key Layout
+ *	 _____	 _____	 _____
+ *	| 	  | | 	  | | 	  |
+ *	|  1  | |  2  | |  3  |
+ *	|_____| |_____| |_____|
+ *	 _____	 _____	 _____
+ *	| 	  | | 	  | | 	  |
+ *	|  4  | |  5  | |  6  |
+ *	|_____| |_____| |_____|
+ *
+ *  Each Layout row below keys. 1,2,3,4,5,6
+ *
+ *  Hold 3 when powering on for DFU Program Mode
+ */
+
+/* clang-format off */
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    /*
+     *   Layer 0 (Media Keys)
+     *	 _____	 _____	 _____
+     *	| 	  | | 	  | | 	  |
+     *	 Stop    VolUp   Play
+     *	|_____| |_____| |_____|
+     *	 _____	 _____	 _____
+     *	| 	  | | 	  | | 	  |
+     *	 Prev    VolDwn  Next
+     *	|_____| |_____| |_____|
+     *
+     */
+    LAYOUT(
+        LT_MSTP, KC_VOLU, KC_MPLY,
+        KC_MPRV, KC_VOLD, KC_MNXT
+    ),
+
+    /*
+     *   Layer 1 (Copy and Paste Macros)
+     *	 _____	 _____	 _____
+     *	| 	  | | 	  | | 	  |
+     *	  ESC    Ctl+Z   CSf+Z
+     *	|_____| |_____| |_____|
+     *	 _____	 _____	 _____
+     *	| 	  | | 	  | | 	  |
+     *	 Ctl+X   Ctl+C   Ctl+V
+     *	|_____| |_____| |_____|
+     *
+     */
+
+    LAYOUT(
+        LT_ESC,  C(KC_Z), C(S(KC_Z)),
+        C(KC_X), C(KC_C), C(KC_V)
+    ),
+
+    /*
+     *   Layer 2 (Spectacles Macros)
+     *	 _LT&__	 _____	 _____
+     *	| Win | | Win | | Win |
+     *	| Next| | Top | |Mximze
+     *	|_1/3_| |_1/2_| |_____|
+     *	 _____	 _____	 _____
+     *	| Win | | Win | | Win |
+     *	| Prev| | Bot | | Next|
+     *	|_1/3_| |_1/2_| |Displ|
+     *
+     */
+    LAYOUT(
+        LT_NXTH,TOPHALF,MAXIMIZ,
+        PRVTHRD,BTMHALF,NXTDSPL
+    ),
+
+    /*
+     *  Layer 3 (Layer Select/Reset)
+     *	 _____	 _____	 _____
+     *	| 	  | | 	  | |     |
+     *	|None | |None | |Reset|
+     *	|_____| |_____| |_____|
+     *	 _____	 _____	 _____
+     *	|Layer| |Layer| |Layer|
+     *  |  0  | |  1  | |  2  |
+     *	|_____| |_____| |_____|
+     *
+     *  Layers 0, 1, and 2 have the top left button held to activate this layer.
+     *  Then press the specific layer to switch to it.
+     *
+     */
+    LAYOUT(
+        _______, _______, RESET,
+        TO(0),   TO(1),   TO(2)
+    )
+
+};
+/* clang-format on */
+
+/*
+ *	Knops Mini LED Numbers:
+ *	 _____	 _____	 _____
+ *	| 	  | | 	  | | 	  |
+ *	|  1  | |  2  | |  3  |    <---
+ *	|_____| |_____| |_____|       |      These LEDs are called 'Switch LEDs'
+ *	 _____	 _____	 _____        |----- To turn on/off these leds, use:
+ *	| 	  | | 	  | | 	  |       |	  set_switch_led( [1-6], [true/false]);
+ *	|  4  | |  5  | |  6  |    <---
+ *	|_____| |_____| |_____|
+ *
+ *	 < 0 >   < 1 >   < 2 >     <---      These front-LEDs are called 'Layer LEDs'
+ *							             To turn one of them on, use:
+ *										 set_layer_led( [0-2] );
+ */
+
+void set_switch_led(int ledId, bool state) {
+    if (state) {
+        switch (ledId) {
+            case 1:
+                PORTD |= (1 << 7);
```

Comment:
These can be simplified to eg.
```suggestion
                writePinHigh(D7);
```